index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="main-container">
+      <div class="header">
+        <div class="logo">
+          <div class="logo-image">
+            <img
+              src="https://th.bing.com/th/id/OIP.jtdO5VDiwFucIyW4UzS9rAHaB-?w=350&h=93&c=7&r=0&o=5&dpr=1.3&pid=1.7"
+              alt=""
+            />
+          </div>
+          <div class="logo-text">Original Trombones</div>
+        </div>
+        <div class="navigation-links">
+          <ul>
+            <li>
+              <a href="#">Features</a>
+            </li>
+            <li>
+              <a href="#">How it Works</a>
+            </li>
+            <li>
+              <a href="#">Pricing</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="hero-section">
+        <div>
+          <p class="hero-section-text">Handcrafted, home-made masterpieces</p>
+          <input
+            class="herosection-input"
+            type="text"
+            name=""
+            id=""
+            placeholder="Enter your Email Address"
+          />
+          <button class="hero-section-button">Get Started</button>
+        </div>
+      </div>
+      <div class="services">
```

Comment:
I would suggest you wrap the services section with **section** tag for more semantic meaning and professionalism

Reply:
Hi, @Samuelwanza!! 👋
- Here you have made a good suggestion with reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better you wrap the services section". 
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.